### DISCUSSION TITLE
Possible to use in a subdirectory?

### DISCUSSION THREAD

**Discussion by Skyfay** [Lissy93/dashy#666] [Help and Support]
Hey, can i use dashy in a subdirectory like:

    location /dashy {
        proxy_pass http://172.1.1.1:80;
    }
?
Thanks for reply :)

**Comment by lissy93** [comment]
For reference, see also: #31,  #193,  #249,  #351,   #543,  #600 and #628

But yeah, it _is possible_, just requires a bit of config on your proxy.
On Dashy's side, all you need to set the `BASE_URL` env var to your new path ([see here](https://github.com/Lissy93/dashy/blob/master/docs/management.md#passing-in-environmental-variables)), and probably also set `routingMode: hash` under [`appConfig`](https://github.com/Lissy93/dashy/blob/master/docs/configuring.md#appconfig-optional).

On your proxy, I'm not sure off the top of my head, but take a look at those other tickets, as well as the example web server configs in the [Management Docs](https://github.com/Lissy93/dashy/blob/master/docs/management.md), which explains all this in more detail.

Hope that helps :)

**Reply by Skyfay** [reply]
> For reference, see also: #31, #193, #249, #351, #543, #600 and #628
> 
> But yeah, it _is possible_, just requires a bit of config on your proxy. On Dashy's side, all you need to set the `BASE_URL` env var to your new path ([see here](https://github.com/Lissy93/dashy/blob/master/docs/management.md#passing-in-environmental-variables)), and probably also set `routingMode: hash` under [`appConfig`](https://github.com/Lissy93/dashy/blob/master/docs/configuring.md#appconfig-optional).
> 
> On your proxy, I'm not sure off the top of my head, but take a look at those other tickets, as well as the example web server configs in the [Management Docs](https://github.com/Lissy93/dashy/blob/master/docs/management.md), which explains all this in more detail.
> 
> Hope that helps :)

It won't work :(
`    location /windows {
      proxy_pass          http://172.19.0.140:80/;
      add_header Access-Control-Allow-Origin *;
}`
` routingMode: hash`
`- BASE_URL=/path`
i get redirected to 404 page...
